presto-main/src/main/java/com/facebook/presto/sql/planner/DistributedExecutionPlanner.java

```diff
@@ -158,13 +174,17 @@ private Visitor(Session session)
                     .map(ExtractDynamicFiltersResult::getDynamicFilters)
                     .orElse(ImmutableSet.of());
 
+            SplitSource splitSource;
             // TODO: use dynamic filters
             if (!dynamicFilters.isEmpty()) {
                 System.out.printf("[DistributedExecutionPlanner] Dynamic filters: %s\n", dynamicFilters);
+                List<ListenableFuture<DynamicFilterDescription>> futuresList
+                        = dynamicFilterDescriptionFutureSupplier.get(session.getQueryId().toString(), dynamicFilters, scan.getAssignments());
```

Comment:
I would rather pass the list of futures, so it can be implemented lazily in the connector